HPC/Gromacs.md

```diff
@@ -1,16 +1,28 @@
-Gromacs is one of the most widely used molecular dynamics software packages. One important way for Gromacs to achieve high performance is to use Modern processors' SIMD (single instruction multiple data) capabilities to increase floating-point performance. Below is the recommended SIMD option for different architectures:
+Gromacs is one of the most widely used molecular dynamics software packages. One important way for Gromacs to achieve high performance is to use Modern processors' SIMD (single instruction multiple data) capabilities to increase floating-point performance. An SVE implementation for the inner kernel has been added to the latest release, Gromacs 2022.4. Below is the recommended SIMD option for different aarch64 architectures:
 * Hpc7g/c7gn (-DGMX_SIMD=ARM_SVE)
 * c6gn (-DGMX_SIMD=ARM_NEON_ASIMD)
-We noticed that on the same Hpc7g instance, which supports both SVE and NEON, choose ARM_SVE as SIMD option gives 10% better performance than ARM_NEON_ASIMD; For FFT, you can use both ArmPL FFT and FFTW library. ArmPL FFT has better performance.
+We noticed that on Hpc7g instance, which supports both SVE and NEON vectorization, choosing ARM_SVE as the SIMD option helped us get 10% better performance than choosing ARM_NEON_ASIMD; For FFT, you can use both ArmPL FFT and FFTW library. ArmPL FFT has better performance.
```

Comment:
Remove 'we noticed',  state exactly what a customer should do as a best practice.